From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alexander Mikhalitsyn <aleksandr.mikhalitsyn@canonical.com>
Date: Sun, 18 Feb 2024 15:43:20 +0100
Subject: [PATCH] confile: unhide lxc_config_parse_arch() helper

Looks safe enough to be available for liblxc users.

Signed-off-by: Alexander Mikhalitsyn <aleksandr.mikhalitsyn@canonical.com>
(cherry picked from commit 42eeffcb05c468fd7b3a90eeda4a3abe9f26844b)
---
 src/lxc/confile.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/lxc/confile.h
+++ b/src/lxc/confile.h
@@ -88,7 +88,7 @@ extern void lxc_config_define_free(struc
  * Parse personality of the container. Returns 0 if personality is valid,
  * negative errno otherwise.
  */
-__hidden extern int lxc_config_parse_arch(const char *arch, signed long *persona);
+extern int lxc_config_parse_arch(const char *arch, signed long *persona);
 
 __hidden extern int lxc_clear_config_item(struct lxc_conf *c, const char *key);
 
